### ISSUE TITLE
Feature Request: option to disable filling of arguments

### ISSUE THREAD

**Issue by justinjhendrick** [atom/autocomplete-plus#846]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description
I would like to be able to autocomplete a function name (in C++) without also getting the templates and argument list.

### Steps to Reproduce

1. define a function `void foo_function(int x, double y) { return x; }`
2. from somewhere else type `foo_f`
3. See `foo_function(int x, double y)` as a suggestion
4. Choose it

**Expected behavior:** 
`unction` is inserted

**Actual behavior:**
`unction(int x, double y)` is inserted

This is especially annoying when I'm renaming a function.

### Versions
atom 1.16.0
autocomplete-plus 2.34.2

### Additional Information
I would like a checkbox in the preferences to control this behavior, please

**Comment by no1hitjam** [comment]
I agree. It's extremely common in JavaScript to reference functions without calling them. In this case, filling in the arguments is a waste of time because you have to delete what autocomplete generates.

In general, I find typing out arguments myself to be much faster than parsing the autocomplete generation of arguments, and deleting and replacing them with my own. This issue needs to be a top priority, as it's taking minutes out of my workflow, and constantly taking me out of the flow of programming.
